cmd/crc/cmd/daemon.go

```diff
@@ -24,7 +25,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var watchdog bool
+
```

Comment:
The commit log could be `Add a mechanism to shutdown the daemon if the parent process dies`, slightly easier to read, and shorter.